app/src/main/java/com/infomaniak/mail/ui/main/thread/DetailedContactBottomSheetDialog.kt

```diff
@@ -29,8 +29,8 @@ import com.infomaniak.mail.utils.notYetImplemented
 
 class DetailedContactBottomSheetDialog : BottomSheetDialogFragment() {
 
-    lateinit var binding: BottomSheetDetailedContactBinding
     private val navigationArgs: DetailedContactBottomSheetDialogArgs by navArgs()
+    lateinit var binding: BottomSheetDetailedContactBinding
```

Comment:
Why isn't it in private ?
I prefer in first line to keep a coherence with the other fragments